op-service/txmgr/cli.go

```diff
@@ -115,6 +119,12 @@ func CLIFlagsWithDefaults(envPrefix string, defaults DefaultFlagValues) []cli.Fl
 			Value:   defaults.SafeAbortNonceTooLowCount,
 			EnvVars: prefixEnvVars("SAFE_ABORT_NONCE_TOO_LOW_COUNT"),
 		},
+		&cli.Uint64Flag{
+			Name:    FeeLimitMultiplierFlagName,
+			Usage:   "The multiplier applied to fee suggestions to put a hard limit on fee increases",
+			Value:   defaults.FeeLimitMultiplier,
+			EnvVars: prefixEnvVars("FEE_LIMIT_MULTIPLIER"),
```

Comment:
```suggestion
			EnvVars: prefixEnvVars("TXMGR_FEE_LIMIT_MULTIPLIER"),
```
We're trying to use the txmgr prefix for new flags that don't need to maintain compatibility.